public/app/features/explore/Graph/ExploreGraph.tsx

```diff
@@ -126,6 +130,8 @@ export function ExploreGraph({
     return applyThresholdsConfig(withGraphStyle, thresholdsStyle, thresholdsConfig);
   }, [fieldConfig, graphStyle, yAxisMaximum, thresholdsConfig, thresholdsStyle]);
 
+  const internalDataLinkSupplier = useExploreInternalDataLinkSupplier(splitOpenFn, timeRange);
```

Comment:
How is this one different from the default implementation that remain in grafana/data? is it the binding to splitOpenFn?

Reply:
Yes, it binds spitOpen (with feature tracking). It also contains some logic for correlations which work only in Explore at the moment. Some of could be moved out (maybe implemented as a macro), but the gist is that it creates special "onClick" action.